modules/delegates.js

```diff
@@ -295,25 +385,23 @@ __private.checkDelegates = function (publicKey, votes, state, cb) {
  * @returns {setImmediateCallback} 
  */
 __private.loadDelegates = function (cb) {
-	var secrets;
 
-	if (library.config.forging.secret) {
-		if (Array.isArray(library.config.forging.secret)) {
-			secrets = library.config.forging.secret;
-		} else {
-			secrets = [library.config.forging.secret];
-		}
-	}
+	var secretsList = library.config.forging.secret;
 
-	if (!secrets || !secrets.length) {
+	if (!secretsList || !secretsList.length || !library.config.forging.force || !library.config.forging.defaultKey) {
```

Comment:
I am not sure about `library.config.forging.force`. @Isabello suggested adding this condition. My understanding is, we only want to load delegates on startup (using default key) for testnet, devnet and `library.config.forging.force` helps in identifying this. For mainnet, we expect the users to hit `/api/delegate/forging/enable` endpoint to activate forging on a node and don't want to load delegates directly from config file.